### PR TITLE
Object model attributes

### PR DESCRIPTION
**Pull request by ericstj**
Add back attributes that were previously on ICommand in desktop.

In order to do this I had to add ValueSerializerAttribute, which lives in System on desktop as well as pull down TypeConverterAttribute.

### CODE REVIEW

File: src/System.ObjectModel/src/System/Windows/Markup/ValueSerializerAttribute.cs [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            {
+                if (_valueSerializerType != null)
+                    return _valueSerializerType.AssemblyQualifiedName;
+                else
+                    return _valueSerializerTypeName;
+            }
+        }
+
+        private Type _valueSerializerType;
+        private string _valueSerializerTypeName;
```
Comment (stephentoub) [inline src/System.ObjectModel/src/System/Windows/Markup/ValueSerializerAttribute.cs]: Nit: could we move these to the top of the class as we have them elsewhere?

Comment (ericstj) [reply src/System.ObjectModel/src/System/Windows/Markup/ValueSerializerAttribute.cs]: Sure.  That bugged me too.

Comment (ericstj) [reply src/System.ObjectModel/src/System/Windows/Markup/ValueSerializerAttribute.cs]: While I'm at it I'll just make them auto-properties.

**Comment by ericstj** [comment]
test OSX x64 Debug Build

**Pull request merged**